ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/PrivxJsonRpcMethodsTest.java

```diff
@@ -76,6 +77,7 @@ public void privxFindPrivacyGroupMethodIsEnabledWhenFlexiblePrivacyGroupIsEnable
 
   @Deprecated
   @Test
+  @Ignore("Ignoring deprecated test.")
```

Comment:
We don't want to ignore this. Until we fully remove the deprecated flexiblePrivacyGroups methods (which won't be for a while), we should continue to test them.